tsl/test/sql/dist_hypertable.sql.in

```diff
@@ -9,7 +9,7 @@
 \o /dev/null
 \ir include/filter_exec.sql
 \ir include/remote_exec.sql
-\ir utils/testsupport.sql
+\ir utils/tsl_debugsupport.sql
```

Comment:
This is the only location the `tsl_debugsupport.sql` file is referenced, so shouldn't it be in the `tsl/` directory then?

Reply:
the utils directory is linked to be available in same location in both tsl and non-tsl tests